testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -182,10 +186,6 @@ Feature: Synchronize products in the products page of the Setup Wizard
     Then I should not see a "Operation not successful" text
     And I should only see success signs in the product list
 
-@scc_credentials
-  Scenario: Trigger a refresh of the products synched from SCC
-    When I execute mgr-sync refresh
```

Comment:
Interesting, so we have two steps doing the same?
If so... should we remove one of them? 🤔